test/widgets/all_channels_test.dart

```diff
@@ -202,21 +203,33 @@ void main() {
       } else {
         check(maybeToggle).isNull();
       }
-
-      check(findInRow(find.byIcon(ZulipIcons.more_horizontal))).findsOne();
     }
   });
 
-  testWidgets('tapping three-dots button opens channel action sheet', (tester) async {
+  testWidgets('open channel action sheet on long press', (tester) async {
     await setupAllChannelsPage(tester, channels: [eg.stream()]);
 
-    await tester.tap(find.byIcon(ZulipIcons.more_horizontal));
-    await tester.pump();
+    await tester.longPress(find.byType(AllChannelsListEntry));
+    await tester.pumpAndSettle();
     await transitionDurationObserver.pumpPastTransition(tester);
```

Comment:
Using pumpAndSettle should be avoidable here, and risks hiding bugs:
https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-using-pumpandsettle

What goes wrong if you leave it at `pump`? That seems like it might point to a bug somewhere.

Reply:
it's actually fine if leave it at `pump()`, thanks for the reference. 

I should probably remove the `pumpAndSettle()` in the `navigate to channel feed on tap` test as well